README.md

```diff
@@ -10,14 +10,32 @@ See https://hugo-theme-beautifulhugo.netlify.app/
 
 Install Hugo and create a new site. See [the Hugo documentation](https://gohugo.io/getting-started/quick-start/) for details.
 
-Add Beautifulhugo:
+### Git Submodule
+
+Add Beautifulhugo as git submodule:
 
     $ git submodule add https://github.com/halogenica/beautifulhugo.git themes/beautifulhugo
 
+### Hugo module
+
+Initialize your site as hugo module:
+
+    $ hugo mod init github.com/USERNAME/SITENAME
+
+Add Beautifulhugo module as a dependency of your site:
+
+    $ hugo mod get github.com/halogenica/beautifulhugo
+
+### Site preview
+
 Copy the content of `exampleSite` at the root of your project:
 
     cp -r themes/beautifulhugo/exampleSite/* . -iv
-    
+
+If you installed Beautifulhugo as hugo module, set your theme in your config file (hugo.toml):
+
+    theme = "github.com/halogenica/beautifulhugo"
```

Comment:
Is it not
```toml
[[module.imports]]
  path = 'github.com/halogenica/beautifulhugo'
```

? (Looking at https://gohugo.io/hugo-modules/use-modules/)

Reply:
> Is it not
> 
> ```toml
> [[module.imports]]
>   path = 'github.com/halogenica/beautifulhugo'
> ```
> 
> ? (Looking at https://gohugo.io/hugo-modules/use-modules/)

Both are valid options.

Your suggestion is more recenty than mine, so your suggestion is now given in `README.md`.